### ISSUE TITLE
Split a track

### ISSUE THREAD

**Issue by khargy** [tkrajina/gpxpy#176]
I might be missing it, but as far as I can tell there's no way to split a track into two tracks...

If I do:

input_gpx.tracks[0].split(0, 0)

That works as expected and my first track get an extra track segment.

but if I do:

input_gpx.split(0, 0, 0)

I expected my first track to be split into two, but I get the same result as the first method, same number of tracks but my first track gets an extra track segment

**Comment by tkrajina** [comment]
Exactly, as the documentation on the `split()` method says:
```
        """
        Splits one of the segments of a track in two parts. If one of the
        split segments is empty it will not be added in the result. The
        segments will be split in place.

        Parameters
        ----------
        track_no : integer
            The index of the track to split
        track_segment_no : integer
            The index of the segment to split
        track_point_no : integer
            The index of the track point in the segment to split
        """
```

**Comment by khargy** [comment]
Sorry about that I wasn't trying to imply the documentation was wrong, I was asking if there was another method I missed that split the track.

**Comment by tkrajina** [comment]
I'm not saying you implied that the documentation is wrong. You asked "I expected my first track to be split into two, but ..." and I replied that, yes, that's how it works. 

Also note that you opened an "issue" for something that isn't an issue -- it's a question. That's the reason why I closed it. Stackoverflow is a better place to ask questions. If (after you searched the source of gpxpy) you feel that a method like the one you wanted should be in the library -- feel free to open a feature request or send a pull request.
